sandbox/apparmor/apparmor.go

```diff
@@ -95,6 +96,10 @@ func setupConfCacheDirs(newrootdir string) {
 		}
 	}
 	SnapConfineAppArmorDir = filepath.Join(dirs.SnapdStateDir(newrootdir), "apparmor", snapConfineDir)
+
+	if err := osutil.EnsureFileState(InternalAbi30File, &osutil.MemoryFileState{Content: []byte(Abi30Text), Mode: 0o644}); err != nil {
```

Comment:
super nitpick: in this project there seems to be preference for the 0644 notation?